src/Apps/Components/AppShell.tsx

```diff
@@ -26,6 +26,15 @@ export const AppShell = props => {
     }
   }, [routeConfig])
 
+  /**
+   * Let our end-to-end tests know that the app is hydrated and ready to go
+   */
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      document.body.setAttribute("data-test-ready", "")
```

Comment:
The format we've been using for data attributes includes [some assignment](https://github.com/artsy/integrity/blob/7dd7aab86a8d4c750d6bd979b76ff193c3ade927/cypress/utils/buyArtworkWithPickup.ts#L42)